packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.styles.ts

```diff
@@ -58,15 +74,35 @@ export function getStyles(props: ITagItemStyleProps): ITagItemStyles {
     close: [
       classNames.close,
       {
-        cursor: 'pointer',
         color: palette.neutralSecondary,
-        fontSize: FontSizes.small,
-        display: 'inline-block',
-        textAlign: 'center',
-        verticalAlign: 'top',
         width: 30,
         height: '100%',
-        flexShrink: 0
+        flex: '0 0 auto',
+        selectors: {
+          ':hover': {
+            background: palette.neutralQuaternaryAlt,
+            color: palette.neutralPrimary
+          }
+        }
+      },
+      selected && {
+        color: palette.white,
+        selectors: {
+          ':hover': {
+            color: palette.white,
+            background: palette.themeDark
+          },
+          [HighContrastSelector]: {
+            color: 'HighlightText'
+          }
+        }
+      },
+      disabled && {
+        selectors: {
+          '.ms-Button-icon': {
```

Comment:
instead of inlining a global class name, you should use an imported button global classname. Is this possible? if we disable global classnames, this wouldn't work.

Reply:
I did try using the `$close` selector in here but with no success. I don't know why in some cases it works and in other cases, it's not.

What would change if I import it from button global classNames and we still have them disabled? Will make the change as it makes sense having it in one place in case it changes, but still, don't see how it can help when disabled.

Reply:
Exported the global button classNames to have it in one place.